yarn.lock

```diff
@@ -12391,6 +12422,11 @@ graphql@^14.6.0, graphql@^14.7.0:
   dependencies:
     iterall "^1.2.2"
 
+graphql@^15.4.0:
+  version "15.4.0"
+  resolved "https://registry.yarnpkg.com/graphql/-/graphql-15.4.0.tgz#e459dea1150da5a106486ba7276518b5295a4347"
+  integrity sha512-EB3zgGchcabbsU9cFe1j+yxdzKQKAbGUWRb13DsrsMN1yyfmmIq+2+L5MqVWcDCE4V89R5AyUOi7sMOGxdsYtA==
+
```

Comment:
I am not sure if it is safe to backport as a patch version because of this. @wardpeet do you know if we actually use the new graphql under the hood with this change (I assume it's from `msw` package)?

Reply:
it's the msw package, it should not impact prod as it's da devDependency. If you have a branch I can test it out on .com

Reply:
Backport PR/branch: https://github.com/gatsbyjs/gatsby/pull/28995

Reply:
I couldn't get .com run with this branch - but it is probably something about windows or my setup. Will try again later when I have a chance. We can publish it in the next patch if things work out well with .com.

Reply:
Could make it work, so should be good to go.